bindings/python/iota_sdk/utils.py

```diff
@@ -220,13 +220,14 @@ def verify_secp256k1_ecdsa_signature(
 
     @staticmethod
     def verify_transaction_semantic(
-            transaction: Transaction, inputs: List[InputSigningData], protocol_parameters: ProtocolParameters, unlocks: Optional[List[Unlock]] = None) -> str:
+            transaction: Transaction, inputs: List[InputSigningData], protocol_parameters: ProtocolParameters, unlocks: Optional[List[Unlock]] = None, mana_rewards: int = 0) -> str:
```

Comment:
Default should be None now
```suggestion
            transaction: Transaction, inputs: List[InputSigningData], protocol_parameters: ProtocolParameters, unlocks: Optional[List[Unlock]] = None, mana_rewards: int = None) -> str:
```